recipes/android-ndk/all/conandata.yml

```diff
@@ -1,4 +1,18 @@
 sources:
+  "r24":
+    url:
+      "Windows":
+        "x86_64":
+          url: "https://dl.google.com/android/repository/android-ndk-r24-windows.zip"
+          sha256: "b2a9fab1481c3c21df0b78608747dde0747b50890134a62a81c983a5250066d6"
+      "Linux":
+        "x86_64":
+          url: "https://dl.google.com/android/repository/android-ndk-r24-linux.zip"
+          sha256: "caac638f060347c9aae994e718ba00bb18413498d8e0ad4e12e1482964032997"
+      "Macos":
+        "x86_64":
```

Comment:
Btw, there is no special download for arm64 Darwin, so I keep x86_64 because it's looks like to be a multi platform download

Reply:
> No clue what's wrong here <img alt="image" width="543" src="https://user-images.githubusercontent.com/3314607/160381042-b480351e-c4cb-4157-b88a-d4baeb3183c4.png">
> 
> <img alt="image" width="738" src="https://user-images.githubusercontent.com/3314607/160381019-2d411c45-adbe-4e2b-a555-34f3ae237908.png">

you need to request access first: see https://github.com/conan-io/conan-center-index/issues/4